### ISSUE TITLE
userspace dynamic background file

### ISSUE THREAD

**Issue by fossfreedom** [libredeb/lightpad#10]
Suggested enhancement.

```
/usr/share/lightpad/background.jpg

/usr/share/lightpad/background.png
```

This limits the ability to change picture to users who have root access.

What I would suggest is that lightpad also looks in ~/.config/lightpad/background.{jpg/png} if location exists.

This allows standard users also the ability to change the background picture.

**Comment by libredeb** [comment]
@fossfreedom yes, it would be a good enhancement to add... I will consider it for future releases. Thank you very much!

**Comment by libredeb** [comment]
This issue are fixed in the commit [#776d962](https://github.com/libredeb/lightpad/commit/776d96246dfff447933718462e05d07728150457).
